server/src/main/java/com/hedera/block/server/Server.java

```diff
@@ -64,8 +63,11 @@ public static void main(final String[] args) {
 
             // Initialize the block storage, cache, and service
             final BlockStorage<BlockStreamServiceGrpcProto.Block> blockStorage = new FileSystemBlockStorage(BLOCKNODE_STORAGE_ROOT_PATH_KEY, config);
+
+            // TODO: Make timeoutThresholdMillis configurable
```

Comment:
we can remove this comment, is already configurable, it probably was bringed on the merge